doc/source/management/data_at_rest_encryption.rst

```diff
@@ -184,6 +184,22 @@ System Variables
 This variable is used to define the location of the
 :ref:`keyring_vault_plugin` configuration file.
 
+.. variable:: encrypt_binlog
+
+  :version 5.7.20-19: Implemented
+  :cli: ``--encrypt-binlog``
+  :dyn: No
+  :scope: Global
+  :vartype: Boolean
+  :default: ``OFF``
+
+This variable turns on binary and relay logs
```

Comment:
This is too skeletal documentation to be useful, and AFAIK it will be fleshed out in DOC-668 ASAP, is this correct?

Maybe add a note there that these are not final docs?